### PR TITLE
AIRFLOW-1399 dont read logfile in memory if remote base isn't specified

### PR DESCRIPTION
**Pull request by NielsZeilemaker**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [X] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-1399

### Description
- [X] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [X] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Small two line change

### Commits
- [X] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Comment by bolkedebruin** [comment]
Is this testable? I would prefer a tested function. Can you correct your commit message? "[AIRFLOW-XX] bla bla" and proper punctuation please.

**Comment by NielsZeilemaker** [comment]
@bolkedebruin I've added two unit-tests, but since the run method wasn't tested before I'm only testing the remote base part

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -469,7 +473,10 @@ def run(args, dag=None):
             DeprecationWarning)
         remote_base = conf.get('core', 'S3_LOG_FOLDER')
 
-    if os.path.exists(filename):
+    if remote_base == 'None':
```
Comment (allisonwang) [inline airflow/bin/cli.py:476]: Why checking equal to string `'None'`? What if the remote base is empty?

Comment (NielsZeilemaker) [reply airflow/bin/cli.py:476]: The if just below this line will check empty strings. So that's covered as well.

**Comment by edgarRd** [comment]
This PR seems to break master, I just pulled latest master and built CI failing on: https://travis-ci.org/edgarRd/incubator-airflow/jobs/256991721#L5421

I can reproduce locally the failure as well. My master seems up to date to latest SHA (6d348903).
@aoen @bolkedebruin @AllisonWang

**Pull request closed without merging**